packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -447,6 +447,7 @@ class AppDomain extends Domain {
   bool isRestartSupported(bool enableHotReload, Device device) =>
       enableHotReload && device.supportsHotMode;
 
+  Future<Null> inProgressHotReload;
```

Comment:
nit: newline after

Perhaps make this private? `_inProgressReload`?

Reply:
Done in ae25a98. C# is private by default, suspect I will keep making this mistake for a while!